SunEngine.Tests/SunEngine.Admin/Managers/CategoriesAdminManagerTests.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Linq;
+using SunEngine.Admin.Managers;
+using SunEngine.Core.Cache.Services;
+using SunEngine.Core.DataBase;
+using SunEngine.Core.Models;
+using SunEngine.Core.Services;
+using SunEngine.Core.Utils.TextProcess;
+using Xunit;
+
+namespace SunEngine.Tests.SunEngine.Admin.Managers
+{
+    public class CategoriesAdminManagerTests
+    {
+        private readonly CategoriesAdminManager catAdminManager;
+        private DataBaseConnection dbConnection;
+
+        public CategoriesAdminManagerTests()
+        {
+            catAdminManager = DefaultCatAdminManager();
+        }
+
+        private CategoriesAdminManager DefaultCatAdminManager()
```

Comment:
It could be worth to rename to create/initialize DefaultCatAdminManager.
Also, in general, it's not recommended to abbreviate.

Reply:
Ok)